### PR TITLE
First example for kafka mgmt

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Running examples
+
+To run examples execute the python modules using the newly aquired ACCESS_TOKEN as follows:
+
+### For kafkamgmt: 
+
+```bash
+ACCESS_TOKEN=`rhoas authtoken`
+python3 examples/kafka_mgmt/get_kafkas_example.py $ACCESS_TOKEN
```
Comment (wtrocki) [inline examples/README.md:19]: Use local variable injection (this way we do not leak token to other commands which is security issue)
```suggestion
ACCESS_TOKEN=`rhoas authtoken` python3 examples/kafka_mgmt/get_kafkas_example.py $ACCESS_TOKEN
```

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+This should return a list of kafkas.
+
+#### Troubleshooting:
+
+One of the common issues one can encounter during use is incomplete commands due to lack of resources (eg: Service Registry artifacts)
+
+This can be resolved by following the steps mentioned in the following file which explains RHOAS "how-to's" as well as a good documentation for first-time users.
+
+>Red Hat OpenShift Service Registry documentation: 
+https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/1630486d-c8d0-45d0-b2de-7fdb52c78499#proc-creating-service-registry-account_getting-started-rhoas-service-registry
```
Comment (wtrocki) [inline examples/README.md:30]: I do not think that would be relevant/important info

```suggestion
```

File: examples/kafka_mgmt/get_kafkas_example.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+configuration = dimakis_test_kafka_mgmt_sdk.Configuration(
+    host = "https://api.openshift.com"
+)
+
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+ACCESS_TOKEN = str(sys.argv[1])
```
Comment (wtrocki) [inline examples/kafka_mgmt/get_kafkas_example.py:17]: use env instead.

File: examples/kafka_mgmt/get_kafkas_example.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+# Configure Bearer authorization (JWT): Bearer
+configuration = dimakis_test_kafka_mgmt_sdk.Configuration(
+    access_token = ACCESS_TOKEN,
+)
+
+
+# Enter a context with an instance of the API client
+with dimakis_test_kafka_mgmt_sdk.ApiClient(configuration) as api_client:
```
Comment (wtrocki) [inline examples/kafka_mgmt/get_kafkas_example.py:26]: I would have assumed that we renamed packages to proper names by now.

File: renovate.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [ "github>redhat-developer/app-services-renovate-config" ],
+  "postUpdateOptions": [
+    "gomodTidy"
+  ],
```
Comment (wtrocki) [inline renovate.json:5]:
```suggestion
```

File: renovate.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+  "extends": [ "github>redhat-developer/app-services-renovate-config" ],
```
Comment (wtrocki) [inline renovate.json:2]: Note: That has no effect as there are no python specific rules. If you intend adding some rules please remember to add them to config.

File: examples/kafka_mgmt/get_kafkas_example.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+ACCESS_TOKEN = str(sys.argv[1])
+
+# Configure Bearer authorization (JWT): Bearer
+configuration = dimakis_test_kafka_mgmt_sdk.Configuration(
+    access_token = ACCESS_TOKEN,
```
Comment (wtrocki) [inline examples/kafka_mgmt/get_kafkas_example.py:21]: Strange to create 2 configuration objects. Can we create single object?

File: examples/kafka_mgmt/get_kafkas_example.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+# Enter a context with an instance of the API client
+with dimakis_test_kafka_mgmt_sdk.ApiClient(configuration) as api_client:
+    # Create an instance of the API class
+    api_instance = default_api.DefaultApi(api_client)
+
+    try:
+        api_response = api_instance.get_kafkas()
+        pprint(api_response)
+    except dimakis_test_kafka_mgmt_sdk.ApiException as e:
```
Comment (wtrocki) [inline examples/kafka_mgmt/get_kafkas_example.py:33]: ApiException would cover cases when we got response from API. 
There are also other exceptions like parsing exception, timeouts etc. 
It is best to have general exception cause as well

File: examples/kafka_mgmt/get_kafkas_example.py (hunk 4 above)
Comment (dimakis) [reply examples/kafka_mgmt/get_kafkas_example.py:26]: so I cannot do that without getting access to vault to get the details to set up the pypi account, that's what I was asking about that again

File: renovate.json (hunk 6 above)
Comment (dimakis) [reply renovate.json:2]: great catch. I will put it on my todo 👍  🙇‍♂️

File: examples/kafka_mgmt/get_kafkas_example.py (hunk 8 above)
Comment (dimakis) [reply examples/kafka_mgmt/get_kafkas_example.py:33]: great catch.

File: examples/kafka_mgmt/get_kafkas_example.py (hunk 7 above)
Comment (dimakis) [reply examples/kafka_mgmt/get_kafkas_example.py]: ah yes. thats the way it was generated. it is strange, you're right.

File: examples/kafka_mgmt/get_kafkas_example.py (hunk 4 above)
Comment (wtrocki) [reply examples/kafka_mgmt/get_kafkas_example.py:23]: Is that a blocker? Can I generate and share tokrn with you and vault?

**Pull request merged**